drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -89,18 +89,20 @@ GLuint RasterizerStorageGLES2::system_fbo = 0;
 #ifndef GLES_OVER_GL
 #define glClearDepth glClearDepthf
 
+#if defined IPHONE_ENABLED || defined ANDROID_ENABLED
 // enable extensions manually for android and ios
 #ifndef UWP_ENABLED
 #include <dlfcn.h> // needed to load extensions
 #endif
+#endif
 
 #ifdef IPHONE_ENABLED
 
 #include <OpenGLES/ES2/glext.h>
 //void *glRenderbufferStorageMultisampleAPPLE;
 //void *glResolveMultisampleFramebufferAPPLE;
 #define glRenderbufferStorageMultisample glRenderbufferStorageMultisampleAPPLE
-#elif defined(ANDROID_ENABLED)
+#elif defined ANDROID_ENABLED || defined HORIZON_ENABLED
```

Comment:
Uh... I don't think this is correct syntax, is it?
I mean, I did test builds, but I thought there had to be parentheses around these...

Reply:
per [cppreference](https://en.cppreference.com/w/cpp/preprocessor/conditional):
> The expression may contain:
> unary operators in form defined identifier or defined (identifier)
So this _is_ valid, but it's best to be consistent.